include/tvm/runtime/threading_backend.h

```diff
@@ -0,0 +1,40 @@
+/*!
+ *  Copyright (c) 2018 by Contributors
+ * \file threading_backend.h
+ * \brief Utilities for manipulating thread pool threads.
+ */
+#ifndef TVM_RUNTIME_THREADING_BACKEND_H_
+#define TVM_RUNTIME_THREADING_BACKEND_H_
+
+#include <functional>
+#include <memory>
+#include <vector>
+
+namespace tvm {
+namespace runtime {
+namespace threading {
+
+class ThreadGroup {
+ public:
+  ThreadGroup();
+  void Launch(std::vector<std::function<void()>> task_callbacks);
+  size_t Size();
```

Comment:
Size is not used by the user of ThreadGroup, so do not have to be in here(can be in the subclass)